pyrogram/methods/messages/get_media_group.py

```diff
@@ -48,10 +48,15 @@ async def get_media_group(
         Raises:
             ValueError: In case the passed message id doesn't belong to a media group.
         """
-        # There can be maximum 10 items in a media group. 
-        messages = await self.get_messages(chat_id, [msg_id for msg_id in range(message_id - 9, message_id + 10)], replies=0)
+        messages = await self.get_messages(
+            chat_id=chat_id,
+            message_ids=[msg_id for msg_id in range(message_id - 9, message_id + 10)],
```

Comment:
This still breaks if you pass a low `message_id` (e.g. 2), as it will try to obtain messages with ids <= 0.

Reply:
When I tested this about 1 month ago, it worked perfectly.

The thing is, if you pass (for example):
_app.get_messages(chat_id=chat_id, [-5, 5])_
it returns a list only with positive message_ids.

So in this case it'd return List of Messages, where message_ids are [from '1' to '5'].
Telegram just doesn't return messages if you pass negative message_id.

Reply:
Ok, what actually breaks is get_messages with a single negative id, which is not applied here.

Reply:
So then we should check if the input `message_id` is not <= 0, right?

Reply:
Dan, please, review the commit.